readme.md

```diff
@@ -1,115 +1,96 @@
 
-2) *Adjust Configuration Files*: Make modifications to `test_filepaths.yaml`, `geography_data.yaml`, and `clean_file_config.yaml` for the kind of clean file to be produced. 
+Note: the filepath and filename for clean files can be adjusted in the [2019 Test Filepaths](https://github.com/cfpb/hmda-test-files/tree/master/2019/configurations/test_filepaths.yaml) YAML configuration file under `clean_filepath` and `clean_filename` keys. 
 
-3) *Create a Clean File*: Run `file.create_files(kind='clean_file')` in the script to produce a clean file saved in a directory specified in `test_filepaths.yaml`. 
+For 2018:
+1. Run `cd 2018/python` to navigate to the python directory. 
+2. Adjust the [2018 Clean File Configuration](https://github.com/cfpb/hmda-test-files/tree/master/2018/configurations/clean_file_config.yaml) for bank name, LEI, Tax ID and number of rows. 
+3. Run `python3 generate_2018_clean_files.py`
+4. The clean test file should appear in a new edits_files directory under `2018/edits_files/clean_files/{Bank Name}/` with the filename `clean_file_{Number or Rows}_{Bank Name}.txt`. 
 
-4) *Create Edit Test Files*: The clean file generated will then be used to create edit test files. Run 
-`file.create_files(kind='error_files')` to produce syntax, validity, and quality edits files for the test institution in filepaths specified in `test_filepaths.yaml`. 
+Note: the filepath and filename for clean files can be adjusted in the [2018 Test Filepaths](https://github.com/cfpb/hmda-test-files/tree/master/2018/configurations/test_filepaths.yaml) YAML configuration file under `clean_filepath` and `clean_filename` keys. 
 
-5) *Validate Quality Edit Test Files*: Ensure that quality edit test files are free of syntax and validity edits. Run `file.validate_quality_edit_file(quality_filename=[quality_file.txt])` in a python script on a quality edit filename.  
+## Generating Test Files
+Test files will fail at least one (and sometimes more than one) edit. These files are used as a double check for the implementation of the HMDA edits in the HMDA Platform for each submission year.
 
-6) *Create a Large File*: Modify `large_file_script_config.yaml` and set the source filepath and source filename to the clean file created. Set the number of rows and/or change the test institution information. Set the filepath and name of the output file. Set the number of rows. Optionally, provide a row by row modification yaml file. Run `large_test_files_script.py` in terminal and create the large file. 
+Creating test files is a two step process. The generation of edit test files requires a clean data file to be present. A clean file is created when generate clean files scripts are run. Running generate test files modifies the data from the clean file and saves it with the name of the edit check that it will fail. If edit files already exist with the same names as those produced by this script, they will be overwrriten.
```

Comment:
Change this to:

`Creating test files is a two step process. The generation of edit test files requires a clean data file to be present. [The steps above](link to clean file generation steps) outline the process to create the clean data files.`